chapters/logging.rst

```diff
@@ -0,0 +1,242 @@
+
+        f = Flow(log_config='logging_cfg.yml').add().add()
+        # If you want to set the configuration for a certain Pod
+        #f = Flow().add(log_config='logging_cfg.yml').add()
+
+In YAML file you can customized the logger, choose the handler you need to monitor and save logs. ``logging_cfg.yml`` can be configured as follows:
+
+.. highlight:: yaml
+.. code-block:: yaml
+
+    handlers:  # enabled handlers, order does not matter
+      - StreamHandler
+      - SysLogHandler
+      - FluentHandler
+    level: INFO  # set verbose level
+    configs:
+      FileHandler:
+        format: '%(asctime)s:{name:>15}@%(process)2d[%(levelname).1s]:%(message)s'
+        output: 'jina-{uptime}.log'
+        formatter: JsonFormatter
+      StreamHandler:
+        format: '{name:>15}@%(process)2d[%(levelname).1s]:%(message)s'
+        formatter: ColorFormatter
+      SysLogHandler:
+        ident: # this will be prepend to all messages
+        format: '{name:>15}@%(process)2d[%(levelname).1s]:%(message)s'
+        host: # when not given then record it locally, /dev/log on linux /var/run/syslog on mac
+        port: # when not given then record it locally,  /dev/log on linux /var/run/syslog on mac
+        formatter: PlainFormatter
+      FluentHandler:
+        # this configuration describes where is the fluentD daemon running and waiting for logs to be emitted.
+        # FluentD then will have its own configuration to forward the messages according to its own syntax
+        # prefix will help fluentD filter data. This will be prepended for FluentD to easily filter incoming messages
+        tag: jina
+        host: 0.0.0.0
+        port: 24224
+        format:
+          host: '%(hostname)s'
+          process: '%(process)s'
+          type: '%(levelname)s'
+
+
+If you want to hide the logs of a certain Pod, you can set ``quiet=True`` in flow like:
+
+.. highlight:: python
+.. code-block:: python
+
+        f = Flow().add(quiet=True).add()
+        with f:
+            f.index(Document())
+
+Or in YAML flow configuration:
+
+.. highlight:: yaml
+.. code-block:: yaml
+
+    !Flow
+    pods:
+      - uses: Pod1.yml
+        quiet: true
+      - uses: Pod2.yml
+
+
+What's more
+-------------
+
+FluentD
+^^^^^^^^^
+`Fluentd <https://github.com/fluent/fluentd>`_ is an open source data collector for unified logging layer.
+
+`Fluentd <https://github.com/fluent/fluentd>`_ is expected to be used as a daemon receiving messages from the Jina logger and forwarding them to specific outputs using its
+output plugins and configurations. 
+ 
+Although fluentd can be configured to forward logs to the user's preferred destinations, Jina offers a default configuration under `/resources` folder which expects a fluentd daemon to be running
+inside every machine running a Jina instance or Pea. Then the default configuration must be adapted to send the logs to the specific server
+where the Flow and the dashboard will be run. (This default behavior will evolve)
```

Comment:
Full stop at the end of line 165

Reply:
Not sure about this one. You mean move content of line 165 to 164.